internal/services/loganalytics/log_analytics_cluster_resource.go

```diff
@@ -187,34 +179,37 @@ func resourceLogAnalyticsClusterUpdate(d *pluginsdk.ResourceData, meta interface
 		return err
 	}
 
-	parameters := clusters.ClusterPatch{}
-	capacityReservation := clusters.ClusterSkuNameEnumCapacityReservation
-	if d.HasChange("size_gb") {
-		parameters.Sku = &clusters.ClusterSku{
-			Capacity: utils.Int64(int64(d.Get("size_gb").(int))),
-			Name:     &capacityReservation,
```

Comment:
is `Name: &capacityReservation,` removed intentionally here?

Reply:
It was! We swapped methods which use a different parameters